package.json

```diff
@@ -8,24 +8,24 @@
     "url": "https://github.com/bendrucker/hapi-raven.git"
   },
   "scripts": {
-    "test": "standard && test-peer-range hapi",
-    "test-main": "tape test.js",
-    "coverage": "100 npm run test-main"
+    "lint": "standard",
```

Comment:
please don't add or re-arrange scripts

Reply:
or remove the peer range tester, just because it happens to match only one major atm

Reply:
ok, I'll revert it to the state before. But your range tester does not work, even if I use a rang in the peer dependencies section of the `package.json`:
```
> standard && test-peer-range hapi

/Users/guischdi/hapi-raven/node_modules/is-comparator-valid/node_modules/semver/semver.js:279

    throw new TypeError('Invalid Version: ' + version);
    ^

TypeError: Invalid Version: 17.0.0
```
Therefor I'll leave it out

Reply:
That's real weird. What node version are you seeing that on locally? Looks like there could be multiple versions of the semver module in the tree and that's causing the error.